project_task_default_stage/README.rst

```diff
@@ -0,0 +1,86 @@
+==========================
+Project Task Default Stage
+==========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproject-lightgray.png?logo=github
+    :target: https://github.com/OCA/project/tree/12.0/project_task_default_stage
+    :alt: OCA/project
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-12-0/project-12-0-project_task_default_stage
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/140/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module restores the ``case_default`` fields to Project Tasks and project
+stage tab, removed in Odoo 9.0 and later.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+After installing this module you can assign stage tasks into project stage tab
+and you can check the default stages for new projects as previously done in
+Odoo 8.0.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/project/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/project/issues/new?body=module:%20project_task_default_stage%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* Carlos Dauden <carlos@incaser.es>
+* Sergio Teruel <sergio@incaser.es>
+* Duc, Dao Dong <duc.dd@komit-consulting.com> (https://komit-consulting.com)
+* `Tecnativa <https://www.tecnativa.com>`_:
+
```

Comment:
Add contributor in the following format:
Ernesto Tejeda <youremail@tecnativa.com> (Tecnativa https://www.tecnativa.com)

Reply:
@emagdalenaC2i email is not mandatory and we prefer to not put it (people spam us asking for free support).